src/org/rascalmpl/uri/URIResolverRegistry.java

```diff
@@ -120,6 +134,57 @@ public Set<String> getRegisteredClassloaderSchemes() {
 		return Collections.unmodifiableSet(classloaderResolvers.keySet());
 	}
 
+	private Object constructService(String name) throws ClassNotFoundException, InstantiationException, IllegalAccessException, IllegalArgumentException, InvocationTargetException, SecurityException {
```

Comment:
Note, this is only an extract of existing code that used to live inside `loadServices`